packages/@react-spectrum/list/test/ListViewDnd.test.js

```diff
@@ -682,7 +682,7 @@ describe('ListView', function () {
       });
 
       act(() => jest.runAllTimers());
-      let droppableButton = await within(droppable).findByLabelText('Drop on Folder 2', {hidden: true});
+      let droppableButton = await within(droppable.parentElement).findByLabelText('Drop on Folder 2', {hidden: true});
```

Comment:
isn't this just the list now? so don't need to access anything via a relationship? There's only going to be a single thing labeled with that text right?

Reply:
ah true, but just wanted to be specific here. Will update this test with the new approach followup